### DISCUSSION TITLE
Encrypted Reply to Anonaddy for decryption

### DISCUSSION THREAD

**Discussion by http403** [anonaddy/anonaddy#476] [Feature Requests]
This is a extension of issue #20. That issue solved the problem when both end have PGP key exchanged, however doesn't work if one side don't have it enabled. This enables mail providers to have a sneek peek on what we recieved as replies usually contains the whole conversation, mainly converstion to customer support in my use case. Also, this will also solve this problem of K-9 automatically encrypting mails when imported `mailer@anonaddy.me` PGP key for signature verification, which from time to time, I forgot to turn off the encryption before reply, and leaks my real email address.

**Comment by willbrowningme** [comment]
Thanks, I'll see if I can add the ability for the server to decrypt replies/sends on the server before sending them on.

**Comment by Torkurson** [comment]
I was actually about to make a post about this too. My use case is very similar to http403's.

BTW, thanks for all the work you've done over the years on this service.

**Comment by willbrowningme** [comment]
I have this working and I'm currently beta testing it on addy.io.

Please [drop me an email](https://addy.io/contact/) with your username if you'd like to help test it out so that I can add you as a beta tester.

See https://github.com/anonaddy/anonaddy/issues/599 for more details and also updates on this feature coming out of beta.
